tools/pot/README.md

```diff
@@ -20,50 +20,72 @@
 ## Introduction
 
 Post-training Optimization Tool (POT) is designed to accelerate the inference of deep learning models by applying
-special methods without model retraining or fine-tuning, like post-training quantization. Therefore, the tool does not
+special methods without model retraining or fine-tuning, e.g. post-training 8-bit quantization. Therefore, the tool does not
 require a training dataset or a pipeline. To apply post-training algorithms from the POT, you need:
 * A floating-point precision model, FP32 or FP16, converted into the OpenVINO&trade; Intermediate Representation (IR) format
 and run on CPU with the OpenVINO&trade;.
-* A representative calibration dataset representing a use case scenario, for example, 300 images.
+* A representative calibration dataset representing a use case scenario, e.g. 300 images. 
+
+Figure below shows the optimization workflow:
+![](docs/images/workflow_simple.png) 
 
 Post-training Optimization Tool provides the following key
 features:
 
 * Two post-training 8-bit quantization algorithms: fast [DefaultQuantization](openvino/tools/pot/algorithms/quantization/default/README.md) and precise [AccuracyAwareQuantization](openvino/tools/pot/algorithms/quantization/accuracy_aware/README.md).
 * Compression for different hardware targets such as CPU and GPU.
 * Multiple domains: Computer Vision, Natural Language Processing, Recommendation Systems, Speech Recognition.
+* [Command-line tool](docs/CLI.md) that provides a simple interface for basic use cases.
 * [API](openvino/tools/pot/api/README.md) that helps to apply optimization methods within a custom inference script written with OpenVINO Python* API.
-* Symmetric and asymmetric quantization schemes. For details, see the [Quantization](openvino/tools/pot/algorithms/quantization/README.md) section.
-* Per-channel quantization for Convolutional and Fully-Connected layers.
+* (Experimental) [Ranger algorithm](@ref pot_ranger_README) for model prodection in safity critical cases.
 
 The tool is aimed to fully automate the model transformation process without a need to change the model on the user's side. For details about 
 the low-precision flow in OpenVINO&trade;, see the [Low Precision Optimization Guide](docs/LowPrecisionOptimizationGuide.md).
 
 For benchmarking results collected for the models optimized with POT tool, see [INT8 vs FP32 Comparison on Select Networks and Platforms](@ref openvino_docs_performance_int8_vs_fp32).
 
-POT is opensourced on GitHub as a part of [https://github.com/openvinotoolkit/openvino](https://github.com/openvinotoolkit/openvino).
+POT is opensourced on GitHub as a part of OpenVINO and available at https://github.com/openvinotoolkit/openvino/tools/pot.
```

Comment:
I think we should move it below, after options with "get started". Otherwise we risk that the user will go to the Low Precision Guide and lost there. Moreover I think that after the options and get started we should explain why and when the user should go to other POT documents because now we have a broken structure and the user doesn't really know where to go from here.

Reply:
Removed this since it is presented in ## See also.